### PR TITLE
Change resolution of temp renderTexture for filter

### PR DESCRIPTION
**Pull request by ivanpopelyshev**
Demo: https://www.pixiplayground.com/#/edit/xwiTwNFC3Q8A5iWPHKkOw

Fixed: https://www.pixiplayground.com/#/edit/S7Om1JRG4jqxbLz0oZ9Lu

### CODE REVIEW

**Comment by ivanpopelyshev** [comment]
Lets discuss the API. I think that if `renderTexture` already has its own `resize`, it should also have `setResolution`.

I have second PR coming right through after that, for more esoteric case when our fullscreen-filter optimization can force weird things.

File: packages/core/src/renderTexture/RenderTexture.js [hunk 1]
```diff
@@ -130,6 +130,24 @@ export default class RenderTexture extends Texture
...
     }
 
+    /**
+     * changes the resolution of baseTexture, but does not change framebuffer size
+     *
+     * @param {number} resolution
+     */
+    setResolution(resolution)
+    {
+        const baseTexture = this.baseTexture;
```
Comment (bigtimebuddy) [inline packages/core/src/renderTexture/RenderTexture.js]:
```suggestion
        const { baseTexture } = this;
```

File: packages/core/src/renderTexture/RenderTexture.js [hunk 2]
```diff
@@ -130,6 +130,24 @@ export default class RenderTexture extends Texture
         this.updateUvs();
     }
 
+    /**
+     * changes the resolution of baseTexture, but does not change framebuffer size
```
Comment (bigtimebuddy) [inline packages/core/src/renderTexture/RenderTexture.js]:
```suggestion
     * Changes the resolution of baseTexture, but does not change framebuffer size.
```

File: packages/core/src/renderTexture/RenderTexture.js [hunk 3]
```diff
@@ -130,6 +130,24 @@ export default class RenderTexture extends Texture
...
+     *
+     * @param {number} resolution
```
Comment (bigtimebuddy) [inline packages/core/src/renderTexture/RenderTexture.js]:
```suggestion
     * @param {number} resolution - The new resolution to apply to RenderTexture
```

**Pull request merged**